cephfs/file_test.go

```diff
@@ -405,3 +405,63 @@ func TestFchown(t *testing.T) {
 	assert.Equal(t, uint32(stats.Sys().(*syscall.Stat_t).Uid), bob)
 	assert.Equal(t, uint32(stats.Sys().(*syscall.Stat_t).Gid), bob)
 }
+
+func TestFstatx(t *testing.T) {
+	fname := "test_fstatx.txt"
+
+	mount := fsConnect(t)
+	defer fsDisconnect(t, mount)
+
+	f, err := mount.Open(fname, os.O_RDWR|os.O_CREATE, 0600)
+	assert.NoError(t, err)
+	assert.NotNil(t, f)
+	assert.NoError(t, f.Close())
+	defer func() { assert.NoError(t, mount.Unlink(fname)) }()
+
+	t.Run("emptyFile", func(t *testing.T) {
```

Comment:
is this for parallelization? In that case it is dangerous to run two tests on the same file that check for the size and write...

Reply:
Is what for parallelization? The t.Run? If so, then no it's not. t.Run is for test case isolation. I want to assert properties on an empty file first, then put some data in there and assert the result is also correct.
Otherwise, I've mis-guessed what you are questioning and you'll have to clarify.